Apps/AdminWebClient/src/ClientApp/src/models/address.ts

```diff
@@ -1,20 +1,17 @@
 // Model that provides a user representation of addresses.
-export default interface Address {
-    // Gets or sets the address.
-    address: string;
-
-    // Gets or sets the second address line.
-    address2?: string;
+export default class Address {
```

Comment:
if this only represents a model that does not provide any methods or calculated fields we use an interface.